shared/telemetry/TelemetryConstants.js

```diff
@@ -30,6 +30,7 @@ export const TelemetryConstants = {
     hardwareKeyStore:
       'Some security features will be unavailable as hardware key store is not available',
     activationCancelled: 'Activation Cancelled',
+    vcsAreTampered: 'VC are tampered',
```

Comment:
Why this change in this commit?

Reply:
Seems like this was a miss from some previous story and since it was an error event it went highly undetected.